packages/reference/src/assets/WrappedAssetCard/AssetCardActions.tsx

```diff
@@ -56,7 +58,12 @@ export function renderAssetInfo(props: { entityFile: File }) {
 
   return [
     <Menu.SectionTitle key="file-section">File info</Menu.SectionTitle>,
-    <Menu.Item key="file-information" className={styles.fileInformation.menuItem} isDisabled>
+    <Menu.Item
+      key="file-information"
+      className={styles.fileInformation.menuItem}
+      isDisabled
+      disabled
```

Comment:
Why is it needed? isDisabled doesn't add actual disabled attribute?

Reply:
Yes 😅

Reply:
It just sets tabIndex of -1

Reply:
So annoying. Same was with NavList.Item 😞

Reply:
Maybe worth pointing this out to the F36 team.